src/components/tools/table-tool/use-measure-column-width.ts

```diff
@@ -1,14 +1,25 @@
 import { useCallback, useRef } from "react";
 import { kMinColumnWidth } from "./table-types";
 import { IAttribute } from "../../../models/data/attribute";
+import { TableContentModelType } from "../../../models/tools/table/table-content";
 
-export const useMeasureColumnWidth = () => {
-  // In the future, these should come from the model rather than being saved in react land
-  const userColumnWidths = useRef<Record<string, number>>({});
+interface IUseMeasureColumnWidth {
+  content: TableContentModelType;
+}
+export const useMeasureColumnWidth = ({ content }: IUseMeasureColumnWidth) => {
+  // The column that is currently being modified
+  const resizeColumn = useRef();
+  // The current width of the column being modified
+  const resizeColumnWidth = useRef();
```

Comment:
As written, it seems like these wouldn't be typed appropriately. Perhaps something like:
```suggestion
  // The id of the column that is currently being modified
  const resizeColumn = useRef<string>();
  // The current width of the column being modified
  const resizeColumnWidth = useRef<number>();
```